examples/QuickStart/QuickStart.ino

```diff
@@ -7,98 +7,52 @@ const byte appKey[16] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 #define debugSerial Serial
 #define loraSerial Serial1
 
-#define debugPrintLn(...) { if (debugSerial) debugSerial.println(__VA_ARGS__); }
-#define debugPrint(...) { if (debugSerial) debugSerial.print(__VA_ARGS__); }
-
 TheThingsNetwork ttn;
 
 void setup() {
   debugSerial.begin(9600);
   loraSerial.begin(57600);
 
   // Wait a maximum of 10s for Serial Monitor
-  uint32_t timeout = millis() + 10000;
-  while (!debugSerial && millis() < timeout);
+  while (!debugSerial && millis() < 10000);
 
   ttn.init(loraSerial, debugSerial);
 
-  debugPrintLn("-- RESET");
+  debugSerial.println("-- STATUS");
+  ttn.showStatus();
```

Comment:
It's good to do `showStatus()` before `join()`, but make sure that in `join()`, you show the DevAddr (see `showStatus()`). That one is set after successful join and is important to know for debugging.